frontend/src/components/blocks/BlocksRow.jsx

```diff
@@ -1,109 +1,118 @@
+import React, { PureComponent } from "react";
+
 import Link from "next/link";
 
 import { Row, Col } from "react-bootstrap";
 
 import Timer from "../utils/Timer";
 
-const BlocksRow = props => (
-  <Link href="/blocks/[hash]" as={`/blocks/${props.block.hash}`}>
-    <a style={{ textDecoration: "none" }}>
-      <Row
-        className={`transaction-row mx-0 ${
-          props.cls === undefined ? "" : props.cls
-        }`}
-      >
-        <Col md="auto" xs="1" className="pr-0">
-          <img src="/static/images/icon-m-block.svg" />
-        </Col>
-        <Col md="auto" xs="1" className="pr-0">
-          <img
-            src="/static/images/icon-m-block.svg"
-            style={{ width: "20px" }}
-          />
-        </Col>
-        <Col md="7" xs="6">
-          <Row>
-            <Col className="transaction-row-title">#{props.block.height}</Col>
-          </Row>
-          <Row>
-            <Col className="transaction-row-text">
+class BlocksRow extends PureComponent {
```

Comment:
Let's use `export default class extends React.PureComponent`

Reply:
fix it